src/components/workshop-code.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import CardIcon from "@/components/ui/card-icon";
+import { cn } from "@/lib/utils";
+import { useState } from "react";
+import { useToast } from "@/components/ui/use-toast";
+
+export const workshopCodeLiteral = "DKEEH";
+
+export function WorkshopCode() {
+  const [codeCopied, setCodeCopied] = useState(false);
+  const { toast } = useToast();
+
+  function copyCode() {
+    navigator.clipboard.writeText(workshopCodeLiteral);
+    setCodeCopied(true);
+    toast({
+      title: `Copied Scrim Code: ${workshopCodeLiteral}`,
```

Comment:
Should have a `duration` of `5000`. Additionally, the toast would look more visually balanced with a description. I think maybe something like this:
![CleanShot 2024-06-25 at 16 44 50@2x](https://github.com/luxdotdev/parsertime/assets/55987186/cc7bfff5-8514-48fb-8eab-812602f09dd5)
```ts
toast({
  title: `Copied Scrim Code to Clipboard!`,
  description: `Code "${workshopCodeLiteral}" has been copied to your clipboard.`,
  duration: 5000,
});
```